pennylane/__init__.py

```diff
@@ -137,7 +137,7 @@ def device(name, *args, **kwargs):
       of qubit-based quantum circuit architectures which allows
       automatic differentiation through the simulation via python's autograd library.
 
-    In addition, additional devices are supported through plugins — see
+    Additional devices are supported through plugins — see
```

Comment:
Much cleaner wording!

Reply:
"In addition, additional" - I bet this was me 🤕 I'm scared to git blame

Reply:
One can never have enough addition!

Reply:
![image](https://user-images.githubusercontent.com/2959003/107255497-54b71500-6a73-11eb-9c2e-2b9561115368.png)

confirmed